### PR TITLE
Add more suggested proguard rules. Better logging when proguard might not be configured properly

### PR DESCRIPTION
**Pull request by drichelson**
Add more suggested proguard rules. Better logging when proguard might not be configured properly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,10 @@ Check out the included example app, or follow things here:
 If you're using ProGuard add these lines to your config:
 
 ```
+-keep class com.launchdarkly.android.** { *; }
+-keep class org.apache.http.** { *; }
```
Comment (jkodumal) [inline README.md]: What inspired these extra keeps?

Comment (drichelson) [inline README.md]: This is the config that our customer reported as fixing their problem.

**Comment by jkodumal** [comment]
👍

**Comment by drichelson** [comment]
[ch225]

**Pull request merged**